.eslintrc.js

```diff
@@ -27,11 +33,17 @@ module.exports = {
         zones: [
           {
             target: ['(public|server)/**/*'],
-            from: ['../../packages/**/*','packages/**/*']
+            from: ['../../packages/**/*','packages/**/*'],
           },
-        ]
-      }
-    ]
+        ],
+      },
+    ],
+    // Add cypress specific rules here
+    'cypress/no-assigning-return-values': 'error',
```

Comment:
wdyt about removing all of the lines with `error` here and addressing each issue?

Reply:
Hi @cwperks, sorry, I was a little bit confusing about what needs to be done here. I think these lines are the rules for cypress related code. For example, `'cypress/no-unnecessary-waiting': 'error'` ensures that if there is `cy.wait(xxx)` in the test body, it will be treated as an error. So by "removing all the lines with `error`', you mean we can remove all of these rules?

Reply:
My bad, I had it the other way around. I thought this was suppressing violations, but its doing the opposite where its enforcing that bad practices are not introduced. Ignore my comment above.